receiver/sqlqueryreceiver/integration_test.go

```diff
@@ -1,8 +1,6 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-//go:build integration
```

Comment:
This breaks Windows CIs, see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36917

Reply:
Oops, I didn't intend to commit that in this PR that was just for local tests. https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36923